CMakeLists.txt

```diff
@@ -665,8 +665,9 @@ if(ICB)
     target_compile_definitions(pyarpack PRIVATE PY_MAJOR_VERSION="3")
     set(pyarpack_HDR ${PROJECT_SOURCE_DIR}/ICB ${PROJECT_SOURCE_DIR}/EXAMPLES/MATRIX_MARKET ${PROJECT_SOURCE_DIR}/EXAMPLES/PYARPACK)
     target_include_directories(pyarpack PUBLIC ${pyarpack_HDR} ${EIGEN3_INCLUDE_DIR} ${Boost_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
-    target_link_libraries(pyarpack BLAS::BLAS LAPACK::LAPACK ${Boost_LIBRARIES} ${PYTHON_LIBRARIES})
+    target_link_libraries(pyarpack PUBLIC arpack ${Boost_LIBRARIES} ${PYTHON_LIBRARIES})
```

Comment:
Should this be PRIVATE?

Reply:
This choice is consistent with the choice for target_include_directories(pyarpack...)